src/components/molecules/DatePickerInput/styles.module.scss

```diff
@@ -0,0 +1,219 @@
+@import 'styles/variables';
+@import 'styles/mixins';
+
+.datePickerContainer {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  .label {
+    @include labelTextStyle;
+    display: inline-block;
+    padding-right: spacing(1.25);
+  }
+
+  .wrapper {
+    display: inline-block;
+    flex-direction: column;
+    position: relative;
+    width: spacing(15.625);
+
+    svg {
+      position: absolute;
+      right: spacing(1);
+      top: spacing(1);
+      pointer-events: none;
+    }
+  }
+
+  input {
+    width: 100%;
+    display: block;
+    appearance: none;
+    background-color: $color-white;
+    border: 1px solid $color-input-border;
+    border-radius: spacing(0.5);
+    color: $color-input-main;
+    @include buttonTextStyle;
+    height: spacing(5);
+    padding: spacing(1);
+
+    &::placeholder {
+      color: $color-input-main;
+    }
+
+    &:focus {
+      outline: none;
+      color: $color-text-dark;
+      border-color: $color-primary;
+    }
+  }
+}
+
```

Comment:
**[non-blocker]**

Not 100% sure, but I think you could have the :disabled selector in the same place, where you have :focus and ::placeholder selectors